### PR TITLE
wpscan: init at 3.4.0

### PR DESCRIPTION
**Pull request by nyanloutre**
###### Motivation for this change

This is my first attempt to bundle a Ruby application, so any comment is welcome

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by c0bw3b** [comment]
@GrahamcOfBorg eval

File: pkgs/tools/security/wpscan/default.nix [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{ bundlerApp, lib }:
+
+bundlerApp {
+  pname = "wpscan";
+  gemdir = ./.;
+  exes = [ "wpscan" ];
+
+  meta = with lib; {
+    description = "black box WordPress vulnerability scanner";
```
Comment (c0bw3b) [inline pkgs/tools/security/wpscan/default.nix]:
```suggestion
    description = "Black box WordPress vulnerability scanner";
```
(nitpick)

File: pkgs/tools/security/wpscan/default.nix [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  gemdir = ./.;
+  exes = [ "wpscan" ];
+
+  meta = with lib; {
+    description = "Black box WordPress vulnerability scanner";
+    homepage    = https://wpscan.org/;
+    license     = licenses.unfreeRedistributable;
+    maintainers = [ maintainers.nyanloutre ];
+    platforms   = platforms.unix;
+};
```
Comment (worldofpeace) [inline pkgs/tools/security/wpscan/default.nix]:
```suggestion
  };
```

**Review by worldofpeace — CHANGES_REQUESTED** [review]
Looking at https://github.com/wpscanteam/wpscan#prerequisites the executable needs to be wrapped with curl.

Something like the snippet in https://github.com/NixOS/nixpkgs/pull/45435 should be sufficient.

**Comment by worldofpeace** [comment]
@GrahamcOfBorg build wpscan

**Comment by worldofpeace** [comment]
Oh yeah unfree.

**Comment by c0bw3b** [comment]
ofBorg can't build unfree pkgs.

**Comment by worldofpeace** [comment]
> ofBorg can't build unfree pkgs.
I know, testing locally.

**Comment by worldofpeace** [comment]
Weird it fails with:
`Unable to identify the wp-content dir, please supply it with --wp-content-dir`

If I don't do that.

**Pull request merged**